entries/b/bibox.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Bibox": {
+    "domain": "bibox.com",
+    "tfa": [
+      "sms",
+      "totp"
```

Comment:
Based on the screenshot provided, it looks like SMS can be used for 2FA, but TOTP can only be used for authorisation: 

> SMS Authentication
> SMS is used to verify **login**, withdrwawal and security reset
> 
> Google Authentication
> Required when withdrawing and changing security settings

Please could you confirm whether TOTP can be used for 2FA at login, or whether it can only be used as authorisation as described in the text above?

Reply:
<img width="1753" alt="2" src="https://user-images.githubusercontent.com/71176569/127499440-10bf8be0-597c-442a-988e-cf58ae0dbab4.png">